sdk/include/reactos/wine/test.h

```diff
@@ -86,6 +87,8 @@ extern const char *wine_dbgstr_longlong( ULONGLONG ll );
 #endif
 static inline const char *debugstr_a( const char *s )  { return wine_dbgstr_an( s, -1 ); }
 static inline const char *debugstr_an( const CHAR *s, intptr_t n ) { return wine_dbgstr_an( s, n ); }
+static inline const char *debugstr_w( const WCHAR *s ) { return wine_dbgstr_wn( s, -1 ); }
+static inline const char *debugstr_wn( const WCHAR *s, int n ) { return wine_dbgstr_wn( s, n ); }
```

Comment:
we have this in `sdk/include/reactos/wine/debug.h`

Reply:
I know that, see the PR description.
But as things are currently done, our wine/test.h doesn't seem capable of including wine/debug.h without errors (last time i tried), and I wanted to keep things close to the wine 3.x state of the things which most of our files use; once we've got past to wine commit c8f338369695d360e021f2cbd82a18b2c0eb1514 for almost all the tests and other synced files, then we could also make the wine/test.h and debug.h able to work together.